packages/cache-manager/README.md

```diff
@@ -332,6 +332,18 @@ await cache.wrap('error', () => {
 
 See unit tests in [`test/wrap.test.ts`](./test/wrap.test.ts) for more information.
 
+## disconnect
+
+`disconnect(key): Promise<void>`
+
+Will disconnect from the relevant store(s).
```

Comment:
Let me know if we should add a small disclaimer about using disconnect only when needed. Depending on the store, for example, redis. It is best practice to re-use connections if possible

Reply:
That would be good to add in launguage about understanding the Keyv driver you are using and when you should use disconnect.

Reply:
I added it.